javaslang-benchmark/src/test/java/javaslang/benchmark/ArrayBenchmark.java

```diff
@@ -0,0 +1,129 @@
+package javaslang.benchmark;
+
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.*;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static javaslang.benchmark.BenchmarkResultAggregator.displayRatios;
+
+public class ArrayBenchmark {
+    public static void main(String... args) throws Exception { /* main is more reliable than a test */
+        final Options opts = new OptionsBuilder()
+                .include(ArrayBenchmark.class.getSimpleName())
+                .shouldDoGC(false)
+                .shouldFailOnError(true)
+                .build();
+
+        final Collection<RunResult> results = new Runner(opts).run();
+        displayRatios(results, "^.*slang.*$");
```

Comment:
only display ratios that have `slang` on either side, i.e. I don't care about `scala_persistent/java_mutable`